package.json

```diff
@@ -39,7 +39,7 @@
     "prettier:fix": "prettier --list-different --write \"**/*{.ts,.tsx,.js,.jsx,.css,.json,.md,.yml}\"",
     "publish": "yarn clean && yarn build && lerna publish",
     "serve": "node scripts/serve.js",
-    "serve:py": "cd app && python -m http.server -b 127.0.0.1",
+    "serve:py": "cd app && python -m http.server -b 127.0.0.1 8000 && echo \nServing at http://127.0.0.1:8000",
```

Comment:
Probably the last `echo` might not be necessary since the previous command should already output the link?

```
Serving HTTP on 127.0.0.1 port 8000 (http://127.0.0.1:8000/) ...
```

Reply:
```suggestion
    "serve:py": "cd app && python -m http.server -b 127.0.0.1 8000",
```